pkg/network/placement_configuration.go

```diff
@@ -1,16 +1,13 @@
 package network
 
 import (
-	"reflect"
-
-	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
 
 	cnao "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/shared"
 )
 
 var (
-	defaultPlacementConfiguration = cnao.PlacementConfiguration{
+	DefaultPlacementConfiguration = cnao.PlacementConfiguration{
```

Comment:
Why are you making a global variable public? It's dangerous, what if somebody changes the value? Global hell.

Please turn it into a function returning the default structure, to avoid mutability.

Reply:
[Done](https://github.com/kubevirt/cluster-network-addons-operator/compare/fae80f26414a0483f7945b258ee2d805475afad6..fa1df96d89d5c6fad893209dad371683d7f80781)